### PR TITLE
Add ublx config to plugin

### PR DESCRIPTION
**Pull request by josephdrichard**
Changes to plugin structure also required to allow sharing data between OnPTPConfigChange and PopulateHwConfig, which is required to write the status to the nodeptpdevice

### CODE REVIEW

File: addons/intel/e810.go [hunk 1]
```diff
@@ -56,21 +68,87 @@ func OnPTPConfigChangeE810(nodeProfile *ptpv1.PtpProfile) error {
 	return nil
 }
 
-func PopulateHwConfigE810(hwconfigs *[]ptpv1.HwConfig) error {
+func AfterRunPTPCommandE810(data *interface{}, nodeProfile *ptpv1.PtpProfile, command string) error {
+	glog.Info("calling onPTPConfigChange for e810 plugin")
```
Comment (jzding) [inline addons/intel/e810.go:72]: calling AfterRunPTPCommandE810

File: addons/intel/e810.go [hunk 2]
```diff
@@ -56,21 +68,87 @@ func OnPTPConfigChangeE810(nodeProfile *ptpv1.PtpProfile) error {
...
+	if data != nil {
+		_data := *data
+		var pluginData *E810PluginData = _data.(*E810PluginData)
+		_pluginData := *pluginData
+		if _pluginData.hwplugins != nil {
+			for _, _hwconfig := range *_pluginData.hwplugins {
+				hwConfig := ptpv1.HwConfig{}
+				hwConfig.DeviceID = "e810"
+				hwConfig.Status = _hwconfig
+				*hwconfigs = append(*hwconfigs, hwConfig)
```
Comment (jzding) [inline addons/intel/e810.go:132]: if there is a hwconfig already exist with DeviceID=="e810", do we replace it or add another resulting in multiple hwconfigs with same DeviceID?

File: pkg/daemon/daemon.go [hunk 3]
```diff
@@ -233,15 +234,18 @@ func (dn *Daemon) applyNodePTPProfiles() error {
...
-						time.Sleep(2 * time.Second)
+						time.Sleep(5 * time.Second)
+						dn.pluginManager.AfterRunPTPCommand(p.nodeProfile, d.Name())
 						d.monitorEvent(clockType, p.configName, p.exitCh, dn.processManager.eventChannel)
 					}
 				}
 			}
 			time.Sleep(1 * time.Second)
 			go p.cmdRun(dn.stdoutToSocket)
+			dn.pluginManager.AfterRunPTPCommand(p.nodeProfile, p.name)
```
Comment (jzding) [inline pkg/daemon/daemon.go:245]: do we need to add a few second delay before running this command?

File: addons/intel/e810.go (hunk 1 above)
Comment (josephdrichard) [reply addons/intel/e810.go]: Fixed

File: pkg/daemon/daemon.go (hunk 3 above)
Comment (josephdrichard) [reply pkg/daemon/daemon.go:245]: We shouldn't.  If necessary for any specific plugin, this can be added inside of the plugin that requires it.

**Comment by josephdrichard** [comment]
submitted-for: CNF-7017

**Pull request merged**